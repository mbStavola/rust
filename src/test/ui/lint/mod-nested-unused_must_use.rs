// aux-build:private-nested-unused_must_use.rs

#![deny(unused_must_use)]

extern crate private_nested_unused_must_use;

use self::private_nested_unused_must_use::{B, C};

fn main() {
    B::new(); // ok: ignores private `must_use` type
    C::new(); //~ ERROR unused `private_nested_unused_must_use::S` in field `s` that must be used
}
