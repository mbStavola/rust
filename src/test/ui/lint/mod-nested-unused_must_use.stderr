error: unused `private_nested_unused_must_use::S` in field `s` that must be used
  --> $DIR/mod-nested-unused_must_use.rs:11:5
   |
LL |     C::new();
   |     ^^^^^^^^^

error: aborting due to previous error

